src/components/ConnectWallet/ConnectedWalletChip.tsx

```diff
@@ -0,0 +1,69 @@
+import { Trans } from '@lingui/macro'
+import { useWeb3React } from '@web3-react/core'
+import { Network } from '@web3-react/network'
+import { Url } from '@web3-react/url'
+import { TextButton } from 'components/Button'
+import Row from 'components/Row'
+import { connections } from 'hooks/connectWeb3/useWeb3React'
+import { WalletDisconnect as WalletDisconnectIcon } from 'icons'
+import Identicon from 'icons/identicon'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+const AccountButton = styled(TextButton)<{ hidden?: boolean }>`
+  filter: none;
+  visibility: ${({ hidden }) => hidden && 'hidden'};
```

Comment:
add a `visible` or `unset` fallback here so it doesn't result in invalid css when hidden isn't set.